### PR TITLE
optimize: optimize Apollo configuration

### LINKED ISSUE #4019: Configuration and registry center related optimizations
- [x] I have searched the [issues](https://github.com/seata/seata/issues) of this repository and believe that this is not a duplicate.

### Ⅰ. Issue Description

Configuration and registry center related optimizations

-  *Properties field is missing

-  *Properties field default value

-  Camel case convert（application.example.yml 、*Properties、*Configuration）

- *Configuration mock test (eg:  getConfig)

-  Improve *Properties UT coverage

### Ⅱ. Describe what happened

  If there is an exception, please attach the exception trace:

```
Just paste your stack trace here!
```

### Ⅳ. How to reproduce it (as minimally and precisely as possible)

1. xxx
2. xxx
3. xxx

### Ⅵ. Environment:

- JDK version :
- Seata version:  
- OS :
- Others:

### PR DESCRIPTION
**Pull request by slievrly**
- [x] I have registered the PR [changes](https://github.com/seata/seata/tree/develop/changes).

### Ⅰ. Describe what this PR did

optimize Apollo configuration

### Ⅱ. Does this pull request fix one issue?

fix #4019

### CODE REVIEW

**Review by a364176773 — APPROVED** [review]
LGTM

File: seata-spring-autoconfigure/seata-spring-autoconfigure-core/pom.xml [hunk 1]
```diff
@@ -11,4 +11,12 @@
 
     <artifactId>seata-spring-autoconfigure-core</artifactId>
     <name>seata-spring-autoconfigure-core ${project.version}</name>
+    <dependencies>
+    <dependency>
```
Comment (jsbxyyx) [inline seata-spring-autoconfigure/seata-spring-autoconfigure-core/pom.xml:15]: please indentation

Comment (slievrly) [reply seata-spring-autoconfigure/seata-spring-autoconfigure-core/pom.xml]: fixed

**Review by xingfudeshi — APPROVED** [review]
LGTM.

**Review by jsbxyyx — APPROVED** [review]
LGTM

**Pull request merged**